hubblestack/hec/obj.py

```diff
@@ -0,0 +1,327 @@
+
+import socket
+import json
+import time
+import copy
+import os
+import hashlib
+
+import certifi
+import urllib3
+
+from . dq import DiskQueue
+
+# fakelogging goes to a file iff specified in environment
+# and otherwise does nothing (for debugging)
+import fakelogging as logging
+log = logging.getLogger('obj')
+
+__version__ = '1.0'
```

Comment:
Does this need versioning independent of hubble?

Reply:
no, probably not. I think originally I was thinking of disk queue versioning that might need something like that.

Reply:
right, it has to do with the hubble-hec/1.0 user agent string more than anything. I could just move that into the user_agent population area and leave it out of globals, but it made sense to me at the time I wrote it; and seems to still knowing about user-agent strings.